javascript/javascript1/week3/CactusIO-interactive.js

```diff
@@ -0,0 +1,41 @@
+// Adding an activity
+let activities = [];
+function addActivity(date, activity, duration) {
+  if (
+    typeof date === "string" &&
+    typeof activity === "string" &&
+    typeof duration === "number"
+  ) {
```

Comment:
Great you have added some extra data validation in to make sure the data is valid!

Reply:
Thanks!!